src/Layout/Navigation/RecentItem.tsx

```diff
@@ -11,26 +11,137 @@
  */
 
 import React from 'react';
-import { Link } from 'react-router-dom';
-import { NavItem } from '@patternfly/react-core';
+import { AlertVariant, Dropdown, DropdownItem, KebabToggle, NavItem } from '@patternfly/react-core';
```

Comment:
I'm not sure if we can do anything with that, but when focus is moved to IDE, left nav bar kebab menu is kept opened, even after I click on IDE:
![Screenshot_20210402_094130](https://user-images.githubusercontent.com/5887312/113388929-c8044580-9397-11eb-8470-34e321712672.png)
![Screenshot_20210402_094139](https://user-images.githubusercontent.com/5887312/113388930-c89cdc00-9397-11eb-8fc5-17857888e016.png)